### PR TITLE
BugFixes-in-RateCard

### PR DESCRIPTION
**Pull request by Akash2172**
## Type of Change

- [x] Bugfix
- [ ] New feature
- [ ] Enhancement
- [ ] Refactoring
- [ ] Dependency updates

### CODE REVIEW

File: Frontend/ui-customer/src/Components/RateCard/View.purs [hunk 1]
```diff
@@ -379,7 +379,7 @@ driverAdditionView push config =
     , imageView
       [ height $ V 100
       , width MATCH_PARENT
-      , imageWithFallback "ny_ic_driver_addition_table,https://assets.juspay.in/beckn/nammayatri/user/images/ny_ic_driver_addition_table.png"
+      , imageWithFallback "ny_ic_driver_addition_table2,https://assets.juspay.in/beckn/nammayatri/user/images/ny_ic_driver_addition_table2.png"
```
Comment (Rohit4997) [inline Frontend/ui-customer/src/Components/RateCard/View.purs:382]: can you remove previous `ny_ic_driver_addition_table`  image if it is not used
Also just replace the image no need to change image name

Comment (witcher-shailesh) [reply Frontend/ui-customer/src/Components/RateCard/View.purs:382]: For now remove the older icon, as we want the image to S3 as well name is fine for now

**Review by witcher-shailesh — CHANGES_REQUESTED** [review]
Added comments

File: Frontend/ui-customer/src/Components/RateCard/View.purs (hunk 1 above)
Comment (kavya-shree-s) [reply Frontend/ui-customer/src/Components/RateCard/View.purs:382]: @witcher-shailesh image pushed to S3 can we overridden with latest image with same name

Comment (SrinivasNaikB) [reply Frontend/ui-customer/src/Components/RateCard/View.purs:382]: Answering on behalf of @Akash2172 
- ny_ic_driver_addition_table,png depicts that driver optional addition of upto 10 is only allowed for trips if distance 5km to 10 km (this was the previously planned Fare Policy and we might anytime go back to that if need be)
- ny_ic_driver_addition_table2,png- depicts a similar information as the earlier version of the image, but it also says drivers are allowed to add upto Rs.10 for less than 5km trips also.

We need to keep both in S3 as there is plan to shift between these two fair policies soon in future. Keeping both can allow us to achieve the shift smoothly with a simple hotpush when needed.

**Review by witcher-shailesh — APPROVED** [review]

**Pull request merged**